.github/workflows/docs-parity-updates.yml

```diff
@@ -63,7 +63,7 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.PRO_ACCESS_TOKEN }}
           REPOSITORY_NAME: localstack-ext
           ARTIFACT_ID: parity-metric-ext-raw-*
-          WORKFLOW: "Build, Test, Push" 
+          WORKFLOW: "AWS / Build, Test, Push" 
```

Comment:
I think to make this more resilient in the future, we might be able to refer to the workflow by its filename (`aws_main.yml`)

At least in the GitHub CLI you can choose the workflow by its name as well as its file name. 

In this case it still wouldn't help (since the file name also changed), but I feel that the file name would change less often than the workflow name.

Reply:
That's a good point, maybe in the future this would make sense, but as you mentioned right now, the filename is basically as stable as the name 😛